plugins/scaffolder/src/next/TemplateWizardPage/Stepper/useTemplateSchema.test.tsx

```diff
@@ -41,6 +41,7 @@ describe('useTemplateSchema', () => {
           description: 'Step 2 Description',
           schema: {
             type: 'object',
+            'ui:layout': 'MyLayout',
```

Comment:
hmm, I'm wondering if it makes sense to build this into the `ui:*` or if we should put it under the `ui:backstage` options instead. Don't really have a strong opinion, but the other `ui:` stuff is basically transparent with the `react-jsonschema-form` library.

Reply:
We could make this field transparent with `react-jsonschema-form` by not using our own name `ui:layout`, we chose this name, but the actual field is called [ui:ObjectFieldTemplate](https://react-jsonschema-form.readthedocs.io/en/latest/advanced-customization/custom-templates/#objectfieldtemplate).

Would dropping `ui:layout` and using the real field name resolve this?  If not, I am happy to move it to `ui:backstage` instead.

Reply:
I think for now maybe we can use `ui:ObjectFieldTemplate` then instead.

Reply:
the field is now called `ui: ui:ObjectFieldTemplate`